lib/examine-eslintrc.js

```diff
@@ -79,6 +79,11 @@ module.exports = function(options) {
             data = JSON.parse(data);
         } catch (e) {
             verboseLog(e);
```

Comment:
I considered adding options here to solve the issue, but it will display a hideous stack trace for the user instead of just notifying the user of the problem.